packages/starlight-llms-txt/types.ts

```diff
@@ -28,6 +28,7 @@ export interface ProjectContext {
 	promote: NonNullable<StarlightLllmsTextOptions['promote']>;
 	demote: NonNullable<StarlightLllmsTextOptions['demote']>;
 	exclude: NonNullable<StarlightLllmsTextOptions['exclude']>;
+	pageSeparator: StarlightLllmsTextOptions['pageSeparator'];
```

Comment:
For consistency we should probably set this as `NonNullable` here too. So the defaults are all set in the same place:

```suggestion
	pageSeparator: NonNullable<StarlightLllmsTextOptions['pageSeparator']>;
```